applications/root.md

```diff
@@ -0,0 +1,367 @@
+
+### Milestone 2 - stated.tech x Polkassembly - Front-End - 56.5k
+
+- **Estimated duration:** 2 month
+- **FTE:**  4
+- **Costs:** 56,500 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can create a preferundum, join a team, edit and update a proposal, send to vote |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 1. | Prototype | Wireframes and prototyping will be done on Figma. |
+| 2. | Module: Root.js // React app | This repo will be treating our front-end for preferendums using React framework. Create a preferendum, join a team, add subjects and alternatives, and vote. From this website, you can first create your preferundum. Whenever someone new wants to participate, quorum is requested to accept their request (or not), which will give them the right to enter the channel, participate, and vote in the preferendum. Root will store the information of each preferundum on IPFS (question of the preferundum, subjects and their alternatives). Then will put the hash on-chain through the pallet preferundum. Through this UI, you can **join** (if the team is not full), you will tokengate your wallet which allows you to join the channel and the team. You can also access a  **list** of all active and passed/rejected preferundums. Root will get back the hash of all preferendums, and get back the text of IPFS and then display it to the interface. You can also create an alternative (subproposal) by adding a new possibility, or cancel your publications. Root will store them on IFPS, add their hashmap on chain and then them it in the UI. |
+
+## Future Plans
+
+Once we've deployed preferendums for small and mid-size teams, we're planning to use Root pallets to enhance treasury mechanisms and develop our logic on several parachains as a standardized governance process. We would most likely use parathreads to deploy our logic seemlessly while remaining a **common good experiment**.
+
+As a Network Public, Root's goal is to become a sovereign network focused on delivering Public Services related to governance and acting as a laboratory for the DotSama ecosystem. We're thus planning to develop several types of preferendums and treasury mechanisms to act as a GaaS network (Governance as a Service).
```

Comment:
GaaS is part of our vision and overall roadmap but is not the subject of this particular proposal. Indeed, it would mainly involve DAOs and Substrate-based app-specific blockchains wanting to benefit from Root's block space that would be sold to these orgs. It would also involve subscriptions for for-profit (mainly web2) companies and DAOs that would want to benefit from a trustless mechanism to access preferendums and several other governance mechanisms, from the creation of a legally compliant organization to role management and decision-making systems developed by Root.

Reply:
Overall, our goal would be to enter the substrate builders program with Root to eventually propose a Common Good Parachain for Statemine that would be governance-oriented.